apps/antalmanac/src/components/Calendar/CalendarToolbar.tsx

```diff
@@ -178,6 +178,7 @@ function SelectSchedulePopover(props: { scheduleNames: string[] }) {
                 variant="outlined"
                 onClick={handleClick}
                 sx={{ minWidth, maxWidth, justifyContent: 'space-between' }}
+                disabled={AppStore.getSkeletonMode()}
```

Comment:
Well, I'm curious if updating the state matters? If Websoc is down, is AntAlmanac going to react to when it's up again?

My current working assumption is that the only way to leave skeleton mode is to refresh the page?

Reply:
That's probably the visible behavior. But because it's part of an observable store, it's best to subscribe to the changes for consistency. i.e. the skeleton mode would otherwise just be a const value that's determined on page load that we know would never change.

Reply:
Aye aye 🫡